### PR TITLE
Fix some typos in JET_SELECTION_LOGIC

### PR DESCRIPTION
**Pull request by supwest**

### CODE REVIEW

File: Comanche055/JET_SELECTION_LOGIC.agc [hunk 1]
```diff
@@ -177,7 +177,7 @@ PYTABLE		OCT	0		# 0	0		0
 		OCT	2421		# +	+		3
 		OCT	2610		# -	+		3
 		OCT	0146		# 0	-		6
-		OCT	2504		# + 	-		6
+		OCT	2504		# + -		6
```
Comment (chrislgarry) [inline Comanche055/JET_SELECTION_LOGIC.agc]: This is not right: http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1042.jpg

**Comment by chrislgarry** [comment]
@supwest  Everything looks good except for the one mistake I commented on. Fix that and I can merge this in. Thanks.

**Comment by supwest** [comment]
Ok, I think it should be good now.

File: Comanche055/JET_SELECTION_LOGIC.agc [hunk 2]
```diff
@@ -470,7 +470,7 @@ ACYJETS		OCT	34360
 # THAN A MINIMUM IMPULSE (14MS), ALL JET CHANNEL COMMANDS WILL BE HELD FIXED FROM THE START OF THE T5 PROGRAM FOR
 # AT LEAST 14MS UNTIL THE INITIALIZATION OF NEW COMMANDS.  MOREOVER, A 14MS ON-TIME WILL BE ADDED TO ANY ROTATIONAL
 # COMMANDS GENERATED BY THE MANUAL CONTROLS OR THE JET SWITCHING LOGIC, AND ALL TRANSLATION COMMANDS WILL BE
-# ACTIVE FOR AT LEAST ONE CYCLE OF THE T5 PROGRAM (.1SEC)
+# ACTIVE FOR ATLEAST ONE CYCLE OF THE T5 PROGRAM (.1SEC)
```
Comment (Nek-) [inline Comanche055/JET_SELECTION_LOGIC.agc]: it was pretty good like that :D

Comment (oznogon) [inline Comanche055/JET_SELECTION_LOGIC.agc]: It is `ATLEAST` on the scanned source of [Comanche 055 page 1050 LR0385](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1050.jpg).

**Comment by oldmud0** [comment]
Squashed it for you.

**Pull request merged**